tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_example_valid_invalid.py

```diff
@@ -384,6 +391,874 @@
             EOFException.UNDEFINED_INSTRUCTION,
             id="jump_jumpdest_fails",
         ),
+        pytest.param(
+            # Check that that code that uses removed opcodes fails (Data 19)
+            Container(
+                name="EOF1I0015",
+                sections=[
+                    Section.Code(
+                        code=Op.JUMPI(3, 1) + Op.JUMPDEST + Op.STOP,
+                        code_inputs=0,
+                        code_outputs=NON_RETURNING_SECTION,
+                        max_stack_height=1,
+                    ),
+                    Section.Data("0xef"),
+                ],
+            ),
+            "ef00010100040200010007040001000080000160016003575B00ef",
+            EOFException.UNDEFINED_INSTRUCTION,
+            id="jumpi_jumpdest_fails",
+        ),
+        pytest.param(
+            # Check that that code that uses removed opcodes fails (Data 20)
+            Container(
+                name="EOF1I0015",
+                sections=[
+                    Section.Code(
+                        code=Op.SELFDESTRUCT(1) + Op.STOP,
+                        code_inputs=0,
+                        code_outputs=NON_RETURNING_SECTION,
+                        max_stack_height=1,
+                    ),
+                    Section.Data("0xef"),
+                ],
+            ),
+            "ef0001010004020001000404000100008000016001ff00ef",
+            EOFException.UNDEFINED_INSTRUCTION,
+            id="selfdestruct_fails",
+        ),
+        pytest.param(
+            # Check that that code that uses removed opcodes fails (Data 21)
+            Container(
+                name="EOF1I0015",
+                sections=[
+                    Section.Code(
+                        code=Op.SELFDESTRUCT(1) + Op.STOP,
```

Comment:
I don't understand the reason why we have two `SELFDESTRUCT` opcode tests here.

Reply:
yes likely a copy paste, since the other test is actually testing data section